src/sagas/wallet.js

```diff
@@ -138,13 +136,15 @@ export function* startWallet(action) {
     dispatch = _dispatch;
   });
 
+  let networkSettings = yield select(getNetworkSettings); 
```

Comment:
Is this mutated anywhere? Can we use `const`?

Reply:
- Declared as const: 8d9bdefbd6bea7a6849747cdeb1aa5cfc0233d2b